cypress/integration/user-journey.js

```diff
@@ -0,0 +1,233 @@
+describe('User journey of login and upload abstracts, visualise then delete upload', () => {
+
+    beforeEach(() => {
+        cy.visit('/');
+        cy.viewport(1920, 1080)
+    });
+
+    it('Lets login, go to results page, check its empty then try to upload an abstract file', () => {
+            cy.visit('/logout');
+            cy.visit('/');
+            cy.get('#side-menu')
+                .contains('Login')
+                .click()
+            cy.get('title')
+                .invoke('text')
+                .should('equal', 'TeMMPo: Login')
+
+            cy.get('#id_username').type(Cypress.env('CREDENTIALS_USR'));
+            cy.get('#id_password').type(Cypress.env('CREDENTIALS_PSW'));
+            cy.get('button[type=submit]').click()
+
+            cy.get('title')
+                .invoke('text')
+                .should('equal', 'TeMMPo: My list')
+
+        // goto results tab and check its empty
+
+            cy.get('#side-menu')
+                .contains('Results')
+                .click()
+
+            cy.get('.page-header')
+                .should('have.text', 'Results')
+
+            cy.get('table')
+                .find('tr')
+                .eq(1)
+                .contains('No data available in table', { matchCase: false })
+
+        // lets go to the search page try to upload an ovid medline abstract file
+
+            cy.get('.controls')
+                .find('a.btn.btn-default')
+                .contains('New search', { matchCase: false })
+                .click()
+
+            cy.get('title')
+                .invoke('text')
+                .should('equal', 'TeMMPo: Search')
+
+            cy.get('.form-group')
+                .find('a.btn.btn-primary')
+                .eq(0)
+                .contains('Upload OVID MEDLINE® formatted abstracts', { matchCase: false })
+                .click()
+
+        // Goes to the ovid medline abstract file upload page
+
+            cy.get('title')
+                .invoke('text')
+                .should('equal', 'TeMMPo: Search Ovid MEDLINE®')
+
+            cy.get('input[type=file]')
+                .selectFile('test-abstract-ovid-test-sample-5.txt')
```

Comment:
Can you reference the original file in the `tests/` directory instead of uploading a new file?

Reply:
> I read that the file needs to be a "path to a file within the project root". i wernt sure which root.

i'll try it and see